src/crimson/os/seastore/cache.h

```diff
@@ -415,6 +448,8 @@ class Cache {
     Transaction &t,
     paddr_t offset,
     extent_len_t length,
+    extent_len_t e_off,
+    extent_len_t e_len,
```

Comment:
IIUC, the original parameters `offset` and `length` are only used to query cache, while the new `e_off` and `e_len` are used to read data from disk, am I right?